.prettierrc

```diff
@@ -0,0 +1,4 @@
+{
+  "semi": false,
+  "singleQuote": true
+}
```

Comment:
I've seen this in `package.json` before in other projects. Do you have a preference? I'm good with this change, just curious of thoughts.

Reply:
Related PR: 
https://github.com/vigetlabs/our-react-way/pull/80